src/mlpack/tests/hoeffding_tree_test.cpp

```diff
@@ -569,6 +569,73 @@ BOOST_AUTO_TEST_CASE(HoeffdingTreeSimpleDatasetTest)
   }
 }
 
+BOOST_AUTO_TEST_CASE(NumDescendantsTest1)
+{
+  // Generate data.
+  arma::mat dataset(3, 500);
+  arma::Row<size_t> labels(500);
+  data::DatasetInfo info(3); // All features are numeric.
+  for (size_t i = 0; i <500; i ++)
+  {
+    dataset(0, i) = mlpack::math::Random();
+    dataset(1, i) = mlpack::math::Random();
+    dataset(2, i) = mlpack::math::Random();
+    labels[i] = 0;
+  }
+
+  // Now train streaming decision tree;
+  typedef HoeffdingTree<GiniImpurity, HoeffdingDoubleNumericSplit> TreeType;
+  TreeType streamTree(info, 3);
+  for (size_t i = 0; i < 500; ++i)
+    streamTree.Train(dataset.col(i), labels[i]);
+
+  BOOST_REQUIRE_EQUAL(streamTree.NumDescendants(), 0); // one label
```

Comment:
Pedantic style issue, please use complete sentences, also you can put the comment above ``BOOST_REQUIRE_EQUAL`` if you run into the 80 columns length limit.